<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UIOK9AL" failures="1" tests="1" name="Test2" time="4.674" errors="0" timestamp="2021-11-12T21:55:29 IST">
  <testcase classname="Framework.Automation.ValidateHeading" name="basePageNavigation" time="0.466">
    <failure type="java.lang.AssertionError" message="expected [Get back to] but found [Get back to growth with the world’s #1 CRM, powered by Customer 360.]">
      <![CDATA[java.lang.AssertionError: expected [Get back to] but found [Get back to growth with the world’s #1 CRM, powered by Customer 360.]
at Framework.Automation.ValidateHeading.basePageNavigation(ValidateHeading.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
</testsuite> <!-- Test2 -->
